src/chat-model.ts

```diff
@@ -646,7 +646,45 @@ ${toolsList}
           const cellType = cell.cell_type;
           const lang = cellType === 'code' ? kernelLang : cellType;
 
-          return `**Cell [${cellInfo.id}] (${cellType}):**\n\`\`\`${lang}\n${code}\n\`\`\``;
+          let outputs = '';
+          if (cellType === 'code' && Array.isArray(cell.outputs)) {
+            outputs = cell.outputs
+              .map(
+                (output: {
+                  output_type: string;
+                  text: any;
+                  traceback: any[];
+                  data: any;
```

Comment:
Is there a way to get real types ? There are a lot of `any` type here, the cell itself is `any`.